develop/dev-guide-use-common-table-expression.md

```diff
@@ -1,26 +1,29 @@
 ---
 title: Common Table Expression
+summary: Introduce the CTE capability of TiDB, which is used to write SQL statements more efficiently.
 ---
 
 # Common Table Expression
 
-Due to the objective complexity of the business, sometimes a single SQL statement as long as 2000 lines is written, which contains a large number of aggregations and multi-level sub-query nesting. Maintaining such SQL is a developer's nightmare.
+In some transaction scenarios, due to application complexity, you may sometimes write a single SQL statement of up to 2,000 lines. The statement probably contains a lot of aggregations and multi-level subquery nesting. Maintaining such an SQL statement can be a developer’s nightmare.
 
-In the previous subsections we have described how to use [views](/develop/dev-guide-use-views.md) to simplify queries and how to use [temporary tables](/develop/dev-guide-use-temporary-tables.md) to cache intermediate query results.
+For more information about simplifying queries, see [Views](/develop/dev-guide-use-views.md). For more details about caching intermediate results, see [Temporary tables](/develop/dev-guide-use-temporary-tables.md) to cache intermediate query results.
```

Comment:
```suggestion
To avoid such a long SQL statement, you can simplify queries by using [Views](/develop/dev-guide-use-views.md) or cache intermediate query results by using [Temporary tables](/develop/dev-guide-use-temporary-tables.md).
```